package.json

```diff
@@ -67,7 +68,7 @@
     "eslint-plugin-import": "^2.29.1",
     "jest": "^29.2.1",
     "metro-react-native-babel-preset": "0.76.5",
-    "prettier": "^2.4.1",
+    "prettier": "^2.8.8",
```

Comment:
This should be done in a separate PR, does not tie back to the scope of the task at hand